src/hardware/input/keyboard_scancodes.cpp

```diff
@@ -444,27 +448,29 @@ std::vector<uint8_t> KEYBOARD_GetScanCode2(const KBD_KEYS key_type,
 	case KBD_printscreen:
 		if (!is_pressed) {
 			// Print Screen gets reported only when released
-			scan_code.resize(6);
-			scan_code[0] = 0xe0;
-			scan_code[1] = 0xf0;
-			scan_code[2] = 0x7c;
-			scan_code[3] = 0xe0;
-			scan_code[4] = 0xf0;
-			scan_code[5] = 0x12;
+			scan_code = {
```

Comment:
How about getting rid of the local `scan_code` variable from all 3 `KEYBOARD_GetScanCode` routines and just doing something like
```c
return { 0xe0, 0xf0, 0x7c, 0xe0, 0xf0, 0x12 };
```
?